server/src/subscription/model.rs

```diff
@@ -45,15 +45,16 @@ impl Subscriptions {
         Ok(Subscriptions(subscriptions))
     }
 
-    pub async fn fetch_by_object<'a, E>(object_id: i32, executor: E) -> Result<Self, sqlx::Error>
-    where
-        E: Executor<'a>,
-    {
+    pub async fn fetch_by_object<'a, A: sqlx::Acquire<'a, Database = sqlx::MySql>>(
+        object_id: i32,
+        acquire_from: A,
+    ) -> Result<Self, sqlx::Error> {
+        let mut transaction = acquire_from.begin().await?;
```

Comment:
With acquire() instead of begin() this won't compile (higher-ranked lifetime error). I think a transaction is fine here but currently lack understanding why it doesn't work with a connection.